php/media/class-gallery.php

```diff
@@ -54,7 +53,6 @@ class Gallery {
 		'carousel_thumbnail_height'         => 64,
 		'carousel_button_shape'             => 'radius',
 		'carousel_thumbnail_selected_style' => 'gradient',
-		'custom_settings'                   => array(),
```

Comment:
@dugajean 

Removing this key generates a notice:
`Undefined index: custom_settings in /var/www/html/wp-content/plugins/cloudinary_wordpress/php/media/class-gallery.php on line 107`

If we don't need it here, do we need it there?